mappings/net/minecraft/network/packet/client/CPacketWorldEvent.mapping

```diff
@@ -1,21 +1,21 @@
-CLASS none/gw net/minecraft/network/packet/client/CPacketEffect
-	FIELD a effectId I
+CLASS none/gw net/minecraft/network/packet/client/CPacketWorldEvent
+	FIELD a eventId I
```

Comment:
Unfortunately we can't really rely on precedent for this because it's all over the place in the other `CPacket...`s. We should probably make an attempt to make all the `CPacket...` use the same field name for event ids, another one that appears to have the same idea behind it is `CPacketBlockEntityUpdate.actionId`, thought it might just be a bad name.